mne_bids/copyfiles.py

```diff
@@ -153,6 +154,77 @@ def copyfile_ctf(src, dest):
                   op.join(dest, bids_folder_name + ext))
 
 
+def copyfile_kit(src, dest, subject_id, session_id,
```

Comment:
you will need to add this to api.rst like the rest of the copyfile_xxx functions